tgui/packages/tgui/interfaces/Cargo.js

```diff
@@ -401,3 +412,58 @@ const CargoCart = (props, context) => {
     </Section>
   );
 };
+
+const CargoHelp = (props, context) => {
+  return (
+    <>
+      <Section title="Department Orders">
+        Each department on the station will order crates from their own personal
+        consoles. These orders are ENTIRELY FREE! They do not come out of
+        cargo&apos;s budget, and rather put the consoles on cooldown. So
+        here&apos;s where you come in: The ordered crates will show up on your
+        supply console, and you need to deliver the crates to the orderers.
+        You&apos;ll actually be paid the full value of the department crate on
+        delivery if the crate was not tampered with, making the system a good
+        source of income.
+        <br />
+        <b>
+          Examine a department order crate to get specific details about where
+          the crate needs to go.
+        </b>
+      </Section>
+      <Section title="MULEbots">
+        MULEbots are slow but loyal delivery bots that will get crates delivered
+        with minimal technician effort required. It is slow, though, and can be
+        tampered with while en route.
+        <br />
+        <b>Setting up a MULEbot is easy:</b><br />
+        <b>1.</b> Drag the crate you want to deliver next to the MULEbot.<br />
```

Comment:
This feels like it should be `<ol>` but then you can't make the numbers bold without some dumb `::marker` stuff, :(